components/MailchimpForm.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable react/no-danger, no-dupe-keys */
+
+import React, { Component } from 'react';
+
+class MailchimpForm extends Component {
+  render() {
+    return (
+      <form
+        action="https://texasjusticeinitiative.us18.list-manage.com/subscribe/post?u=fd262cb4a5fc0bafb38da2e22&amp;id=2663621fac"
+        method="post"
+        target="_blank"
+      >
+        <input
+          className="contact-form__field contact-form__field--medium"
+          style={{ 'margin-bottom': '0.5em' }}
+          type="text"
+          placeholder="First name"
+          name="FNAME"
+          required
+        />
+        <input
+          className="contact-form__field contact-form__field--medium"
+          style={{ 'margin-bottom': '0.5em' }}
+          type="email"
+          placeholder="Email"
+          name="EMAIL"
+          required
+        />
+        <br />
+        <button type="submit" className="btn btn--primary" style={{ 'margin-top': '0.5em' }}>
```

Comment:
I don't remember off-hand if we have setup styles for a `btn--secondary` class, but it may be a good time to do that or setup a class for buttons in the sidebar.

Feel free to do that here, or we can tackle that in a separate issue.

Reply:
Good point. Let's save that one for another PR. I opened https://github.com/texas-justice-initiative/website-nextjs/issues/136.